api/DHSC.FingertipsNext.Api/Program.cs

```diff
@@ -71,4 +75,35 @@ private static void RegisterModules(IServiceCollection services, ConfigurationMa
             module?.RegisterConfiguration(configuration);
         });
     }
+
+    private static void RegisterDbContext(this IServiceCollection services, IConfiguration configuration)
+    {
+        // why don't we just store the connection string as the env variable instead of the bits?
+
+        const string DbNameEnvironmentVariable = "DB_NAME";
```

Comment:
Regarding your recent message - I agree injecting the entire connection string would be preferable to bringing in individual parameters.

Reply:
On further thought constructing and validating the connection string in docker seems more difficult than it's worth at this point.